src/LightPanels/LightPanels.less

```diff
@@ -95,6 +95,7 @@
 
 		> .offscreen {
 			visibility: hidden;
+			opacity: 0;
```

Comment:
Aha! I missed that case. Thanks for informing!
But still have question.. Isn't it enough to just replace visibility: hidden to opacity: 0? I'm just curious...
If so we can remove visibility style. I referred table in this link.
http://stackoverflow.com/questions/272360/does-opacity0-have-exactly-the-same-effect-as-visibilityhidden